src/index.html

```diff
@@ -13,6 +13,40 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--1">
+      <div class="stars star"></div>
+      <div class="stars star"></div>
+      <div class="stars star"></div>
+      <div class="stars star"></div>
+      <div class="stars star"></div>
```

Comment:
The `star` class should be modified to `stars__star` to adhere to the BEM methodology. Each star should be a child element of the `stars` block, following the BEM convention.